styx-api-service/src/main/java/com/spotify/styx/api/SchedulerProxyResource.java

```diff
@@ -82,8 +95,11 @@ private CompletionStage<Response<ByteString>> proxyToScheduler(String path, Requ
             .newBuilder();
     ImmutableSortedMap.copyOf(rc.request().parameters()).forEach((name, values) ->
         values.forEach(value -> builder.addQueryParameter(name, value)));
+    return client.send(withHost(withRequestId(rc.request().withUri(builder.build().toString()))));
```

Comment:
want to replace any of these headers? @honnix 

```x-calling-service=edgeproxyinternal, x-forwarded-for=172.21.52.67, user-agent=Styx Client 2.1.166, via=HTTP/2 edgeproxy} ```

Reply:
I think they should be fine.